tests/functional/object/rgw/test_obc_quota.py

```
@@ -121,18 +123,467 @@ def test_rgw_obc_quota(
+                    pattern=f"post-rm-{label}-",
+                    file_dir=test_dir,
+                    target=f"s3://{name}",
+                    amount=1,
+                    s3_obj=obc_obj,
+                    ignore_error=False,
+                )
+            except CommandFailed:
+                assert False, (
+                    f"Bucket {name} ({label}) should be writable after "
+                    "removing objects but writes failed!!"
+                )
```

Ruff (0.16.0)

[warning] Assertion always fails, replace with `pytest.fail()`

(PT015)